pkg/api/helpers.go

```diff
@@ -178,3 +179,53 @@ func LoadBalancerStatusDeepCopy(lb *LoadBalancerStatus) *LoadBalancerStatus {
 	}
 	return c
 }
+
+// GetAccessModesAsString returns a string representation of an array of access modes.
+// modes, when present, are always in the same order: RWO,ROX,RWX.
+func GetAccessModesAsString(modes []PersistentVolumeAccessMode) string {
+	modesAsString := ""
+
+	if contains(modes, ReadWriteOnce) {
+		appendAccessMode(&modesAsString, "RWO")
```

Comment:
Mean to put that comment above :)

The only place I'm using these constants (outside of tests) is here in helpers for serialization to/from a string.   The binder uses the string for indexing.  The CLI uses the string, too.